bazel/BUILD

```diff
@@ -27,8 +27,13 @@ config_setting(
 )
 
 config_setting(
-    name = "engine_wamr",
-    values = {"define": "engine=wamr"},
+    name = "engine_wamr_interp",
+    values = {"define": "engine=wamr-interp"},
```

Comment:
We could consider leaving it as `wamr`, though I doubt anybody is using this directly... But I don't feel strongly about this, it's just an observation.

Reply:
like ?

```
config_setting(
    name = "engine_wamr",
    values = {"define": "engine=wamr-interp"},
)
```

Reply:
No opinion on my end--leaving it as-is SGTM.